central/complianceoperator/v2/pipelines/complianceoperatorinfo/pipeline.go

```diff
@@ -80,13 +80,15 @@ func (s *pipelineImpl) Run(ctx context.Context, clusterID string, msg *central.M
 		operatorErrors = append(operatorErrors, fmt.Sprintf("Compliance operator is not ready. Only %d pods out of desired %d are ready.", readyPods, desiredPods))
 	}
 
-	// we support only newer versions of compliance operator
-	complianceOperatorVersion, err := semver.NewVersion(operatorInfo.GetVersion())
-	if complianceOperatorVersion == nil || err != nil {
-		log.Error(errors.Wrapf(err, "parsing compliance operator version: %q", operatorInfo.GetVersion()))
-		operatorErrors = append(operatorErrors, fmt.Sprintf("The installed compliance operator version %q is invalid.", operatorInfo.GetVersion()))
-	} else if complianceOperatorVersion.LessThan(env.ComplianceMinimalSupportedVersion.VersionSetting()) {
-		operatorErrors = append(operatorErrors, fmt.Sprintf("The installed compliance operator version %q is unsupported. The minimum required version is %q.", complianceOperatorVersion.String(), env.ComplianceMinimalSupportedVersion.VersionSetting().String()))
+	if operatorInfo.GetVersion() != "" {
```

Comment:
Why do we get this message at all when CO is not installed?
Is it possible to detect it being not installed and stop processing early?
What if CO is installed, but due to some bug the version field is not populated?